ShearTests/StatusMenuTests.swift

```diff
@@ -9,23 +9,23 @@
 import XCTest
 
 @testable import Shear
- 
+
 class StatusMenuTests: XCTestCase {
     var statusMenu: StatusMenu!
-
+    
     override func setUp() {
         super.setUp()
         
         statusMenu = StatusMenu()
     }
-
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/789b229ffaaa71902e329054d9d22803831cd48f/ShearTests/StatusMenuTests.swift#L21"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>StatusMenuTests.swift:21</code></del></td>
    </tr>
  </tbody>
</table>